### ISSUE TITLE
Make the free algebra know it is graded

### ISSUE THREAD

**Issue by tscrim** [sagemath/sage#34406]
<div id="comment:0"></div>

```
sage: R.<x,y> = FreeAlgebra(QQ)
sage: R.category()
Category of algebras with basis over Rational Field
```

Component: **algebra**

_Issue created by migration from https://trac.sagemath.org/ticket/34406_

**Comment by mantepse** [comment]
I think this is fixed by #37199 (@tscrim wrote "closes #34406", but github doesn't seem to recognise this), right?

**Comment by tscrim** [comment]
Yes, it should have been closed.
